src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedBackingFieldSymbol.cs

```diff
@@ -53,6 +49,14 @@ public override ImmutableArray<Location> Locations
             }
         }
 
+        internal override bool EmitDynamicAttributeIfDynamic
```

Comment:
This is an odd name. Why do we need "IfDynamic"? Perhaps better name would be SuppressDynamicAttribute.